Empty newsfeed matches empty performance style

benortiz opened the pull request:
If there were no news items, the no content view was only a plain
paragraph that flowed outside of the layout. This restructures the HTML
inherit styles from Foundation and used by the performance page when
there is no content there.

https://github.com/calraijintaiko/caltaiko/issues/19

![image](https://cloud.githubusercontent.com/assets/681189/14305647/3d642376-fb73-11e5-9e6c-20ade8f8081a.png)

benortiz commented:
I expected to find this in `views/articles/newsfeed.html.haml`, but instead found it in this file, `_thumbnail_feed.html.haml`. Where is `newsfeed.html.haml` actually used?

benortiz commented:
@jgraycar PS how'd I do on PR and commit and branch?? LOL

jgraycar commented:
@benortiz nice, looks good to me! Yeah I think you're right, `_news_feed.html.haml` seems to be leftover and not in use anywhere anymore. Go ahead and take it out, or rename `_thumbnail_feed` to it if you think it's more intuitive.

Branch name and PR look good hahaha. The commit message looks good also! The only thing I'd mention is that I've been trying to follow the guidelines [here](http://chris.beams.io/posts/git-commit/), and the subject line could be reworded a little to match the imperative mood rule. It's totally chill though, I don't do that either half the time :stuck_out_tongue: 

But yeah, overall looks really good! Go ahead and make that one file change and I'll close and merge :+1:

benortiz commented:
@jgraycar AWESOME ARTICLE. I went ahead and removed/renamed the files, fixed up the commit message and yeah, I think it's good to go!

When you're ready, let me know and I'll show you a cool way to do merges without merge messages :)

benortiz commented:
@jgraycar also, your code is much more understandable/readable omg thank you.

jgraycar commented:
Awesome, looks good! 'Tis ready to :ship: 

Haha sweet! Is is using `rebase`? I used it instead of merge for a while until I just started using the merge button on GitHub lol :stuck_out_tongue: But ya, probably a good habit to get back into.

Wooo haha thanks! Probably also because what's going on is so much simpler tho lol

jgraycar commented:
Closes #67

benortiz commented:
ah you already knew >< but yeah, I actually found it easier than having to open up and work in github lol, yay my first commit in years!!
